### PR TITLE
Install torchvision in release docker

### PR DESCRIPTION
**Pull request by lsy323**
Follow up PR to #6756

Only CI testing docker is covered in #6756, cover release docker in this PR.

Release docker is built based on https://github.com/pytorch/xla/blob/master/infra/ansible/Dockerfile.

### CODE REVIEW

**Review by ManfeiBai — APPROVED** [review]
Thanks! LGTM

**Review by JackCaoG — APPROVED** [review]

File: infra/ansible/Dockerfile [hunk 1]
```diff
@@ -20,8 +20,7 @@ ARG ansible_vars
 RUN ansible-playbook -vvv playbook.yaml -e "stage=release" -e "${ansible_vars}" --tags "install_deps"
 
 WORKDIR /tmp/wheels
-COPY --from=build /src/pytorch/dist/*.whl ./
-COPY --from=build /src/pytorch/xla/dist/*.whl ./
+COPY --from=build /dist/*.whl ./
```
Comment (vanbasten23) [inline infra/ansible/Dockerfile:23]: does it mean we don't install the torch wheel and torch_xla wheel?

Comment (lsy323) [reply infra/ansible/Dockerfile:23]: All whls should be installed. I found this from the e2e test dockerfile

Comment (vanbasten23) [reply infra/ansible/Dockerfile:23]: Interesting

**Review by vanbasten23 — APPROVED** [review]

**Review by will-cromar — APPROVED** [review]

File: infra/ansible/Dockerfile (hunk 1 above)
Comment (lsy323) [reply infra/ansible/Dockerfile:23]: @vanbasten23 FYI it's from https://github.com/pytorch/xla/blob/master/infra/ansible/e2e_tests.Dockerfile#L27-L32

**Pull request merged**